src/clientversion.h

```diff
@@ -38,7 +38,7 @@
 #define DO_STRINGIZE(X) #X
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The " PACKAGE_NAME " Developers"
```

Comment:
There is no legal entity "The Bitcoin Core developers", it is just a term used to indicate that each developer retains his own copyright. Replacing the package name in it doesn't change that.